atom/browser/native_browser_view.cc

```diff
@@ -7,14 +7,23 @@
 #include "atom/browser/native_browser_view.h"
 
 #include "atom/browser/api/atom_api_web_contents.h"
-#include "brightray/browser/inspectable_web_contents_view.h"
+#include "brightray/browser/inspectable_web_contents.h"
 
 namespace atom {
 
 NativeBrowserView::NativeBrowserView(
-    brightray::InspectableWebContentsView* web_contents_view)
-    : web_contents_view_(web_contents_view) {}
+    brightray::InspectableWebContents* inspectable_web_contents)
+    : inspectable_web_contents_(inspectable_web_contents) {}
 
 NativeBrowserView::~NativeBrowserView() {}
 
+brightray::InspectableWebContentsView*
+NativeBrowserView::GetInspectableWebContentsView() {
+  return inspectable_web_contents_->GetView();
```

Comment:
Can you check the availability of `inspectable_web_contents_` before using it. Its possible for the `webContents` to get destroyed before the view.

Reply:
I think the existing code already had similar issues. I'll work on fixing it in a separate PR, but for the purposes of fixing the draggable region bug, I think we should go ahead and merge this as-is.